### PR TITLE
Add configuration for target EP_AGORA

### PR DESCRIPTION
**Pull request by AGlass0fMilk**
Enables building this project for Embedded Planet's EP_AGORA target.

### CODE REVIEW

**Comment by AGlass0fMilk** [comment]
@LDong-Arm Any update on this?

File: mbed_app.json [hunk 1]
```diff
@@ -30,6 +30,23 @@
             "nsapi.default-wifi-security": "WPA_WPA2",
             "nsapi.default-wifi-ssid": "\"SSID\"",
             "nsapi.default-wifi-password": "\"PASSWORD\""
+        },
+        "EP_AGORA": {
```
Comment (LDong-Arm) [inline mbed_app.json:34]: Does the example pick up cellular by default on this target? I mean the interface returned by `NetworkInterface::get_default_instance()` in `main.cpp`.

**Review by LDong-Arm — COMMENTED** [review]
@AGlass0fMilk Thanks for the contribution. This config can serve as an example of how to configure the application to use cellular (we have WiFi and Ethernet already).

As we certainly can't list every supported target in `mbed_app.json`, we can extend the README to instruct users to base their configuration on the one you added, if they need to use another cellular-available target.

@evedon Given that we have limited access to hardware to test anything during WFH, shall we just accept EP_AGORA as a "reference config" for cellular targets?

File: mbed_app.json (hunk 1 above)
Comment (LDong-Arm) [reply mbed_app.json:34]: If we are not certain, we can add
```
"target.network-default-interface-type": "CELLULAR",
```

Comment (evedon) [reply mbed_app.json:34]: It is the default for this target, see https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/TARGET_EP_AGORA/mbed_lib.json/#L17

**Review by evedon — APPROVED** [review]

**Comment by evedon** [comment]
Code is okay but the PR should be made tp `development` branch

**Review by evedon — CHANGES_REQUESTED** [review]
Make a PR to `development` instead

**Comment by LDong-Arm** [comment]
> Make a PR to `development` instead

I changed the target branch to "development" with GitHub UI, diff still looks the same.

**Review by evedon — APPROVED** [review]

**Pull request merged**